lib/engine/config/game/g_18_al.rb

```diff
@@ -697,7 +697,10 @@ module G18AL
          "tiles":[
             "yellow"
          ],
-         "operating_rounds": 1
+         "operating_rounds": 1,
+         "status":[
```

Comment:
these statuses seem like they are just descriptions, should we do the refactor you're talking about?

Reply:
Have now done the refactoring.